components/reddit/sources/new-hot-post-on-a-subreddit/new-hot-post-on-a-subreddit.js

```diff
@@ -0,0 +1,184 @@
+const reddit = require("../../reddit.app.js");
+
+module.exports = {
+  key: "new-hot-spot-on-a-subreddit",
+  name: "New hot spot on a subreddit",
+  description:
+    "Emits an event each time a new host post is added to the top 10 a subreddit.",
```

Comment:
Small changes

"Emits an event each time a new **hot** post is added to the top 10 **items in** a subreddit."